src/main/java/dev/snowdrop/release/model/Release.java

```diff
@@ -114,16 +148,14 @@ public void setTest(boolean test) {
                 final int len = suffix > 0 ? version.length() - suffix : version.length();
                 final var expectedVersion = pom.getVersion();
                 if (!this.version.regionMatches(0, expectedVersion, 0, len)) {
-                    errors.add(String.format("'%s' release version doesn't match '%s' version in associated POM", this.version,
-                        expectedVersion));
+                    errors.add(String.format("'%s' release version doesn't match '%s' version in associated POM",
+                            this.version, expectedVersion));
                 }
             }
         }
-        
-        // validate schedule
-        if (schedule == null) {
-            errors.add("missing schedule");
-        } else {
+
+        // validate schedule only if it not null
+        if (schedule != null) {
```

Comment:
I'd rather keep the previous check because now, if schedule is `null` for some reason, the code will proceed without error when it really should fail.

Reply:
my latest commit addresses this